modules/machineset-azure-capacity-reservation.adoc

```diff
@@ -11,9 +11,8 @@ endif::[]
 [id="machineset-azure-capacity-reservation_{context}"]
 = Configuring Capacity Reservation by using machine sets
 
-//Note: This is in 4.16.3, and might be backported through 4.14.z at a later date.
-//Earlier versions of OCP should get an update .z version here. OCP 4.17 docs should use {product-version}.
-{product-title} version 4.16.3 and later supports on-demand Capacity Reservation with Capacity Reservation groups on {azure-full} clusters.
+//If backported to earlier versions, must replace {product-version} with the correct .z
+{product-title} version {product-version} and later supports on-demand Capacity Reservation with Capacity Reservation groups on {azure-full} clusters.
```

Comment:
Because we are talking about multiple versions ({product-version} and later), should this be _support_?

 ```suggestion
{product-title} version {product-version} and later support on-demand Capacity Reservation with Capacity Reservation groups on {azure-full} clusters.
```